alpha/alpha.ino

```diff
@@ -16,6 +16,16 @@ void setup() {
   // disable power to microphone
   TS_HAL.power_set_mic(false);
 
+  // set internal charging current to recommended 100mA 
+  // default is 100mA
+  TS_HAL.power_set_charging_current(CURRENT_100MA);
+
+  // default is 4200mV
+  TS_HAL.power_set_charging_voltage(VOLTAGE_4200MV);
```

Comment:
lets decrease this to 4.15v

Is there also an api to set the low voltage cutoff? It should be set to 3.2v

Reply:
I'll make another PR for that